packages/nimble-components/src/table/tests/table.pageobject.ts

```diff
@@ -3,6 +3,7 @@ import type { TableHeader } from '../components/header';
 import type { TableRecord } from '../types';
```

Comment:
Could we write unit tests for this new code? You previously had tests, but now that the logic is more complex, there aren't any tests. If we stick with the `mousein`/`mouseout` event approach, I'd think you could emit those events within a unit test to verify that an appropriate `title` attribute gets added/removed.

Reply:
Turned out to be fairly easy. Added three tests.